pipelines/skylab/smartseq2_single_nucleus/mouse_paired_example.json

```diff
@@ -0,0 +1,11 @@
+{
+  "SmartSeq2SingleNucleus.genome_ref_fasta": "gs://hca-dcp-sc-pipelines-test-data/referenceData/references/modified_GRCm38.primary_assembly.genome.fa",
```

Comment:
Is this bucket public? These should be updated to point to the google public references when the files are there.

Reply:
We need this to be our bucket, since we do not have the references on Google public bucket yet and the the smart tests need these inputs. I will make a new PR once the files are moved and update these json files.